plugins/omemo/src/ui/contact_details_dialog.vala

```diff
@@ -81,7 +81,7 @@ public class ContactDetailsDialog : Gtk.Dialog {
             own = true;
             own_id = plugin.db.identity.row_with(plugin.db.identity.account_id, account.id)[plugin.db.identity.device_id];
 
-            automatically_accept_new_descr.label = _("When you add new encryption keys to your account, automatically accept them.");
+            automatically_accept_new_descr.label = _("When I add new encryption keys to my account, automatically accept them.");
```

Comment:
That's because of the unusual mode of the sentence in the first place and I didn't want to change it completely.

Your suggestion sounds good, but I'd stick with "encryption keys" instead of "device keys". One reason is to clarify what the keys are for (I assume the average user has an idea what encryption is, but not necessarily what a device key is), the other is that "device" isn't redundantly repeated then.

Reply:
Good point. Let's go with "New encryption keys from your other devices will be accepted automatically.", as you suggested, then.